checkov/terraform/checks/data/base_cloudsplaining_data_iam_check.py

```diff
@@ -0,0 +1,30 @@
+from abc import ABC
+from typing import Dict, List, Any
+
+from cloudsplaining.scan.policy_document import PolicyDocument
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.data.base_check import BaseDataCheck
+from checkov.terraform.checks.utils.base_cloudsplaining_iam_scanner import BaseTerraformCloudsplainingIAMScanner
+from checkov.terraform.checks.utils.iam_terraform_document_to_policy_converter import (
+    convert_terraform_conf_to_iam_policy,
+)
+
+
+class BaseTerraformCloudsplainingDataIAMCheck(BaseDataCheck, BaseTerraformCloudsplainingIAMScanner, ABC):
```

Comment:
not sure, why you subclass `ABC` here? there are no abstract annotations used.

Reply:
Actually we also need to implement `cloudformation_analysis` which is only in the check itself